Changelog.md

```diff
@@ -1,7 +1,13 @@
 ## 2.4.3 - unreleased
 
+### Added
+
+-   Pre-bundled Modem Trace Databases v1.3.6, v1.3.6-FOTA, v1.3.7, v1.3.7-FOTA,
+    v2.0.1, v2.0.1-FOTA, v2.0.2 and v2.0.2-FOTA.
+
 ### Fixed
 
+-   Modem Trace Database dropdown filters out unsupported DB for nRF9151.
```

Comment:
```suggestion
-   Issue where the Modem Trace Database's drop-down would filter out an unsupported database for nRF9151.
```
How is filtering out an _unsupported_ db an issue? Sounds logical to me to filter out something that is not supported.